phac-nml/irida-next
# Samples Table: Fixes preserving page size when searching

## What does this PR do and why?
Persisting the page size during sample filtering.
Fixes #736.

## Screenshots or screen recordings
![image](https://github.com/user-attachments/assets/75d810f2-d838-4006-bae5-35cdd225b83a)
![image](https://github.com/user-attachments/assets/e9361bae-2d3f-4ff2-804b-84131a44b2e7)

## How to set up and validate locally
1. Navigate to a group or project samples page.
2. Select a page size other than the default.
3. Filter the samples.
4. Verify the page size remain unchanged.

## PR acceptance checklist
This checklist encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.
